cookbooks/bcpc-hadoop/recipes/tez.rb

```diff
@@ -5,59 +5,59 @@
 #
 ::Chef::Recipe.send(:include, Bcpc_Hadoop::Helper)
 
-package hwx_pkg_str('tez', node[:bcpc][:hadoop][:distribution][:release]) do
+package hwx_pkg_str('tez', node['bcpc']['hadoop']['distribution']['release']) do
   action :install
 end
 
 directory "/etc/tez/conf.#{node.chef_environment}" do
-  owner "root"
-  group "root"
-  mode 00755
+  owner 'root'
+  group 'root'
+  mode 0o0755
```

Comment:
https://github.com/bbatsov/ruby-style-guide#numeric-literal-prefixes
and the rubocop style doc:
https://github.com/bbatsov/rubocop/blob/076ac7ceb03a9079116fd3a13086884f01292368/manual/cops_style.md#stylenumericliteralprefix